types/setting.go

```diff
@@ -255,6 +258,18 @@ var (
 		Default:     "300",
 	}
 
+	SettingDefinitionFailedBackupTTL = SettingDefinition{
+		DisplayName: "Failed Backup Time to Live",
+		Description: "In seconds. This setting determines how long Longhorn will keep the backup resource that was failed. Set to 0 to disable the auto-deletion.\n" +
+			"Failed backups will be checked and cleaned up during backupstore polling which is controlled by **BackupstorePollInterval** setting. " +
+			"Disabling **Backupstore Poll Interval** also means to disable failed backup auto-deletion.\n\n",
+		Category: SettingCategoryBackup,
+		Type:     SettingTypeInt,
+		Required: true,
+		ReadOnly: false,
+		Default:  "180",
```

Comment:
Just want to reconfirm TTL's purpose and the default value.
As you mentioned, the purpose of the TTL is to let users be aware of the failed backup, and user can check the resources and etc. According to the purpose, 180 seconds seems pretty short. Is bigger value better?

Reply:
This was discussed as well. I think the longer value should be more reasonable. How about 1 day?

Reply:
Yeah, 1 day is reasonable if we would keep failed backups for users.
The original design is that enabling auto-deletion means deleting failed backup ASAP because failed backup is meaningless if users enabled the auto-deletion by themselves.
Now auto-deletion is on by default, and we have the TTL for failed backups.

Modified and changed the unit from `seconds` to `minutes`. Thanks.